filters/accesslog/disable_test.go

```diff
@@ -0,0 +1,62 @@
+package accesslog
+
+import (
+	"testing"
+
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/filters/filtertest"
+)
+
+func TestEnabled(t *testing.T) {
+	const state = "false"
+
+	f, err := NewAccessLogDisabled().CreateFilter([]interface{}{state})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var ctx filtertest.Context
+	ctx.FStateBag = make(map[string]interface{})
+
+	f.Request(&ctx)
+	bag := ctx.StateBag()
+	if bag[AccessLogDisabledKey] != false {
+		t.Error("failed to set access log state")
+	}
+}
+
+func TestDisabled(t *testing.T) {
```

Comment:
If you like you can rewrite all these tests to table driven tests.
https://opensource.zalando.com/skipper/development/#writing-tests shows how to do table driven tests, which will shorten the code.

Reply:
👍